x-pack/test/api_integration/apis/ml/data_frame_analytics/get.ts

```diff
@@ -0,0 +1,240 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+import { USER } from '../../../../functional/services/machine_learning/security_common';
+
+const COMMON_HEADERS = {
+  'kbn-xsrf': 'some-xsrf-token',
+};
+
+export default ({ getService }: FtrProviderContext) => {
+  const esArchiver = getService('esArchiver');
+  const supertest = getService('supertestWithoutAuth');
+  const ml = getService('ml');
+
+  const jobId = `bm_${Date.now()}`;
+
+  async function createJobs() {
+    const mockJobConfigs = [
+      {
+        id: `${jobId}_1`,
+        description:
+          "Classification job based on 'ft_bank_marketing' dataset with dependentVariable 'y' and trainingPercent '20'",
+        source: {
+          index: ['ft_bank_marketing'],
+          query: {
+            match_all: {},
+          },
+        },
+        dest: {
+          index: `user-bm_${jobId}_1`,
+          results_field: 'ml',
+        },
+        analysis: {
+          classification: {
+            dependent_variable: 'y',
+            training_percent: 20,
+          },
+        },
+        analyzed_fields: {
+          includes: [],
+          excludes: [],
+        },
+        model_memory_limit: '350mb',
+      },
+      {
+        id: `${jobId}_2`,
+        description:
+          "Regression job based on 'ft_bank_marketing' dataset with dependentVariable 'duration' and trainingPercent '20'",
+        source: {
+          index: ['ft_bank_marketing'],
+          query: {
+            match_all: {},
+          },
+        },
+        dest: {
+          index: `user-bm_${jobId}_2`,
+          results_field: 'ml',
+        },
+        analysis: {
+          regression: {
+            dependent_variable: 'duration',
+            training_percent: 20,
+          },
+        },
+        analyzed_fields: {
+          includes: [],
+          excludes: [],
+        },
+        model_memory_limit: '350mb',
+      },
+    ];
+
+    for (const jobConfig of mockJobConfigs) {
+      await ml.api.createDataFrameAnalyticsJob(jobConfig);
+    }
+  }
+
+  describe('GetDataFrameAnalytics', () => {
```

Comment:
You have a nested describe with the same name. Could be `GET DataFrameAnalytics`

Reply:
Updated in 2d6a82732ea98cfd50ddb77bb3c09ee29ca6bb63